src/dashboard/dashboard_state.rs

```diff
@@ -0,0 +1,26 @@
+pub struct DashboardState {
```

Comment:
The files `state`, `manager` and `events` are already inside the `dashboard` module, so let's remove the `dashboard` prefix from them.